Automattic/woocommerce-services
# Adds JS test infrastructure

Fixes #1746 

To test:
1. Run `npm install` to get new dependencies
1. Run `npm run test-client`
1. See something like this:
<img width="357" alt="image" src="https://user-images.githubusercontent.com/6209518/67723628-d9bf0d80-f999-11e9-85d9-7ed1153ca87c.png">

I believe all infrastructure and dependency issues are resolved on this PR. I'll open issues for fixing the broken tests and adding a Travis config for these tests.
